chemicaltoolbox/osra/osra.xml

```diff
@@ -44,6 +44,11 @@ osra
             <param name="oformat" value="sdf"/>
             <output name="outfile" ftype="sdf" file="osra_on_CID2244.sdf"/>
         </test>
+        <!-- <test>
```

Comment:
For some reason osra fails when I upload a png (not just in Galaxy, also on the command line). I need to look into it further.

Reply:
Ok, don't look to deep. It will end up in compiling hell. I remember know huge problems with the conda package. All good. Lets remove the png feature for now.